src/LiveComponent/tests/Fixtures/Kernel.php

```diff
@@ -90,6 +90,7 @@ protected function configureContainer(ContainerConfigurator $c): void
             'secrets' => false,
             'session' => ['storage_factory_id' => 'session.storage.factory.mock_file'],
             'http_method_override' => false,
+            'property_info' => ['enabled' => true],
```

Comment:
This was needed?

Reply:
Yes otherwise we don't have access to the phpdoc info